BUILD.md

```diff
@@ -33,7 +33,7 @@ To build all the images simply run:
 Pass version files to the `build` script to build specific versions:
```

Comment:
Whoops. I'll accept this as a separate PR.

Reply:
Removed from this PR